### ISSUE TITLE
StrawberryShake result has empty Extensions

### ISSUE THREAD

**Issue by nightroman** [ChilliCream/graphql-platform#4956]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

The property `Extensions` of `ExecuteAsync` result seems to be always empty even if a GraphQL server returns extension data.

### Steps to reproduce

1. Use the attached solution
2. Build and run

### Actual results

StrawberryShake has no extension data:

```
StrawberryShake Extensions.Count = 0
HttpClient extensions = {
  "requestId": "de6db3fd-3013-45c6-a38d-c58e72d99110"
}
```

### Expected results

StrawberryShake has some extension data:

```
StrawberryShake Extensions.Count = 1
HttpClient extensions = {
  "requestId": "de6db3fd-3013-45c6-a38d-c58e72d99110"
}
```

### Additional Context?

[StrawberryShakeEmptyExtensions.zip](https://github.com/ChilliCream/hotchocolate/files/8451147/StrawberryShakeEmptyExtensions.zip)

### Product

Strawberry Shake

### Version

12.7.0

**Comment by nightroman** [comment]
Could you please reopen the issue? It's still active. Or should I open another one with a link to this?
@tobias-tengler ?

**Comment by nightroman** [comment]
same problem in 12.15.0

**Comment by anton-gogolev** [comment]
Doesn't seem to work with StrawberryShake.Server 13.0.5 either.

**Comment by nightroman** [comment]
@anton-gogolev Yep, same here.

**Comment by nightroman** [comment]
Fixed in 13.2.
